sbnanaobj/StandardRecord/classes_def.xml

```diff
@@ -62,12 +62,13 @@
    <version ClassVersion="10" checksum="1169960296"/>
   </class>
 
-  <class name="caf::SRTrackTruth" ClassVersion="10">
+  <class name="caf::SRTrackTruth" ClassVersion="11">
+   <version ClassVersion="11" checksum="3541008994"/>
    <version ClassVersion="10" checksum="2876855736"/>
   </class>
 
-  <class name="caf::ParticleMatch" ClassVersion="10">
```

Comment:
Is this old definition not required for backwards compatibility? i.e. does root complain if you try and open an old file with a `ParticleMatch` without this being defined?

Reply:
Agree, but the problem is I don't think you can write this without a `ParticleMatch` class actually existing somewhere (the class got renamed to SRParticleMatch` as part of this). On NOvA we had a "morgue" directory full of old data products that we needed to generate dictionary entries for, but shouldn't otherwise be used. You should definitely test what happens if you try to open an old file using this version of the code.

Reply:
Will take a look at this now

Reply:
Yeah this is definitely an issue. I've played around with this a little bit but haven't got anything working yet Will come back to this as soon as I have a free moment and try and work something out.